src/Symfony/Bridge/Doctrine/Form/ChoiceList/IdReader.php

```diff
@@ -95,10 +95,11 @@ public function getIdValue($object)
         }
 
         if (!$this->om->contains($object)) {
-            throw new RuntimeException(
-                'Entities passed to the choice field must be managed. Maybe '.
-                'persist them in the entity manager?'
-            );
+            throw new RuntimeException(sprintf(
+                'Entity of type "%s" passed to the choice field must be managed.'.
+                'Maybe you forget to persist it in the entity manager?',
+                get_class($object)
+            ));
```

Comment:
Can you fix the CS by having everything on one line? Thanks.

Reply:
Done. BTW, since this method is public, do you think should I add a check before using `get_class()` on `$object`?

Reply:
Not sure it's worth it. If one is calling the method with a non-object, they are on their own :)